src/lib/sql/sql_translator.hpp

```diff
@@ -203,12 +203,13 @@ class SQLTranslator final {
 
   static std::string _trim_meta_table_name(const std::string& name);
 
+  // The current LQP might not be cacheable if it involves a meta table
+  bool cacheable{true};
```

Comment:
See PR discussion.

Reply:
> Solving the subquery involving non-cacheable metatables problem by making SQLTranslator's _cacheable a public member

Why does it have to be public? Where are you accessing it from the outside?

Reply:
The SubqueryTranslator is another SQLTranslator, isn't it?

Reply:
So?

> A private member of a class is only accessible to the members and friends of that class, regardless of whether the members are on the same or different instances

Am I missing something?

Reply:
No, but I did, apparently.